src/core/files.py

```diff
@@ -751,12 +751,15 @@ def zip_files(files, article_specific=False):
     os.makedirs(_dir, 0o775)
 
     for file in files:
-        if article_specific and file.article_id:
-            folder_name = '{id} - {title}'.format(id=file.article_id, title=strip_tags(file.article.title))
-            article_dir = os.path.join(_dir, folder_name)
-            if not os.path.exists(article_dir):
-                os.makedirs(article_dir, 0o775)
-            shutil.copy(file.self_article_path(), article_dir)
+        if file.article_id:
```

Comment:
I  think we should still rename this to `zip_article_files` for clarity. I call `zip_files` and pass it a list of non article files, it will return an empty zip, no? I would make it error if a file without an article_id is passed as well, but this might uncover hidden bugs at runtime

Reply:
> I think we should still rename this function to `zip_article_files` for clarity. I call `zip_files` and pass it a list of non article files, it will return an empty zip, no? I would make it error if a file without an article_id is passed as well, but this might uncover hidden bugs at runtime

This is also used for zip downloading review files which have article_ids though aren't loaded into article specific folders for the zip.

Reply:
So those are also article files, the problem is that it will now ignore files without an article_id, but that is only clear after you follow all the if statements in the function, not at all clear from the name